glide.lock

```diff
@@ -0,0 +1,772 @@
+hash: d6a07b4be7b543818b3d224f003e2d59ab0835fd44c939051aee8eb819f0cfed
+updated: 2019-01-31T14:47:19.822803131-05:00
+imports:
+- name: bitbucket.org/ww/goautoneg
+  version: 75cd24fc2f2c2a2088577d12123ddee5f54e0675
+- name: github.com/Azure/go-ansiterm
+  version: d6e3b3328b783f23731bc4d058875b0371ff8109
+  subpackages:
+  - winterm
+- name: github.com/beorn7/perks
+  version: 3ac7bf7a47d159a033b107610db8a1b6575507a4
+  subpackages:
+  - quantile
+- name: github.com/certifi/gocertifi
+  version: ee1a9a0726d2ae45f54118cac878c990d4016ded
+- name: github.com/coreos/etcd
+  version: 420a452267a7ce45b3fcbed04d54030d69964fc1
+  subpackages:
```

Comment:
so this is etcd 3.2.24 openshift 4.0 is using 3.3.x. (3.3.10) Now k8s 1.11 brought in the v3.3.x etcd client I am just wondering if we should bump this as well?

https://github.com/kubernetes/kubernetes/issues/61326#issuecomment-374737257

Reply:
well no that didn't make it until 1.13 https://github.com/kubernetes/kubernetes/pull/69322 nmd.